### DISCUSSION TITLE
Custom artwork for RadioBrowser to get a nice view for the media-control card or get album cover for now playing songs in RadioBrowser

### DISCUSSION THREAD

**Discussion by Vergil365** [music-assistant/support#2808] [Feature requests and ideas]
The title says it all, it would be nice to get a nice view für die media-control card on the dashboard, right now the radiostation from RadioBrowser have none or pixalated pictures which both ugly.
So a custom picture for a radiostation or even better, but not that easy i guess, get automatically album cover for every song played of a radiostation.

**Comment by beejones** [comment]
I have this issue when using music assistant's RadioBrowser in the home assistant app but not when I use it in the browser. In the browser I see the radio station icons. I love the idea to show in real time what the other stations are playing.
